packages/repack/src/utils/__tests__/getModulePaths.test.ts

```diff
@@ -0,0 +1,59 @@
+import { getModulePaths } from '../getModulePaths';
+
+describe('getModulePaths', () => {
+  const packages = [
+    'react-native',
+    'react-native-windows',
+    'react-native-macos',
+    'react-native-tvos',
+    '@callstack/react-native-visionos',
+    '@react-native/core',
+    '@babel/core',
+    '@types/react',
+    '@expo/vector-icons',
+    'socket.io',
+  ];
+
+  const paths = getModulePaths(packages);
+
+  // Classic paths are even-indexed
+  const classicPaths = paths.filter((_, index) => index % 2 === 0);
+  // Exotic paths are odd-indexed
+  const exoticPaths = paths.filter((_, index) => index % 2 !== 0);
```

Comment:
@jbroma Let me know what you think. Now that the `getModulePaths` function accepts an array of deps (and returns a flattened array of regexes) I had to filter classic and exotic paths based on if they're odd or even-indexed.